frontEnd/tickets-app/src/components/App/App.js

```diff
@@ -0,0 +1,16 @@
+import Header from "../Header/Header";
+import Main from "../Main/Main";
+import Footer from "../Footer/Footer";
+import React, { Fragment } from "react";
+
+const App = () => {
+  return (
+    <Fragment>
+      <Header />
+      <Main></Main>
```

Comment:
write `<Main />`

if children of a component are empty, you should use shorthand for that. See:

`<Main></Main>` = `</Main>` 
and the second one is better